ui/src/experiment/ExperimentsRouter.js

```diff
@@ -43,12 +45,26 @@ const RemoteRouter = ({ projectId }) => {
       ? "Failed to load Experiment Engine"
       : "Loading Experiment Engine ...";
     return <FallbackView text={text} />;
+  } else if (!configReady && !configFailed) {
```

Comment:
Can we eliminate this empty check? I think re-writing this way may be cleaner (we move the LoadDynamicScript component up).

```javascript
 if (!ready || failed) {
    const text = failed ? "Failed to load Experiment Engine" : "Loading Experiment Engine ...";
    return <FallbackView text={text} />;
  } else if (!!defaultExperimentEngine.config && !configReady) {
    return configFailed
               ? <FallbackView text={"Failed to load Experiment Engine Config"} />
               :  (<LoadDynamicScript /><FallbackView text={"Loading Experiment Engine Config..."} />);
            // You may need to wrap the 2 components in the else part : in a react Fragment if this is not allowed
}

return (
    <React.Suspense
   ....
// as it was before
```